fcl/caf/cafmakerjob_icarus_sce_genie.fcl

```diff
@@ -0,0 +1,7 @@
+#include "cafmakerjob_icarus_sce.fcl"
+
+services.RandomNumberGenerator: {}
+
+physics.runprod: [ @sequence::caf_ana_sce_evtw_sequence, genieweight, cafmaker ]
```

Comment:
This is the line I was talking about.

Here is a quick fix, but I don't know if this is best or if you want to define a new sequence in `cafmaker_defs.fcl`.
```suggestion
physics.runprod: [ @sequence::caf_ana_sce_sequence, rns, genieweight, cafmaker ]
```